opencti-platform/opencti-graphql/src/modules/publicDashboard/publicDashboard-domain.ts

```diff
@@ -42,16 +41,18 @@ import {
 } from '../../domain/stixCoreObject';
 import { ABSTRACT_STIX_CORE_OBJECT } from '../../schema/general';
 import { findAll as stixRelationships, stixRelationshipsDistribution, stixRelationshipsMultiTimeSeries, stixRelationshipsNumber } from '../../domain/stixRelationship';
-import { bookmarks } from '../../domain/user';
+import { bookmarks, computeAvailableMarkings } from '../../domain/user';
 import { daysAgo } from '../../utils/format';
 import { isStixCoreObject } from '../../schema/stixCoreObject';
 import { ES_MAX_CONCURRENCY } from '../../database/engine';
+import { getMaxMarkings } from '../../domain/settings';
 
 export const findById = (
   context: AuthContext,
   user: AuthUser,
   id: string,
 ) => {
+  logApp.info('[OPENCTI] Public dashboard - trying to fetch public dashboard with id', { id });
```

Comment:
I guess these were logs for dev ? or do we want them in prod ?

Reply:
Indeed, the api is findByUriKey. I added a log on both functions, I can remove this one if you think it is not relevant